### ISSUE TITLE
blame for all lines in file

### ISSUE THREAD

**Issue by sa1dai** [zielu/GitToolBox#256]
Can I have blame not for current line at caret, but for all lines in file?

**Comment by zielu** [comment]
Hi, there is standard IDE annotate action for that but it is showing less information than inline blame.
I will leave it as feature request.

**Comment by sa1dai** [comment]
Yeah, but I just think that it should not be hard to add feature like this.
